WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m

```diff
@@ -1032,7 +1034,8 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     detailController.readerViewStyle = self.readerViewStyle;
     [self.navigationController pushViewController:detailController animated:YES];
 
-    [WPAnalytics track:WPAnalyticsStatReaderOpenedArticle];
+    NSString *detailType = (self.readerViewStyle == ReaderViewStyleNormal) ? ReaderDetailTypeNormal : ReaderDetailTypePreviewSite;
+    [WPAnalytics track:WPAnalyticsStatReaderOpenedArticle withProperties:@{@"post-detail-type":detailType}];
```

Comment:
Seems odd to have two out of three of these strings as constants. Why not the third?